src/hero.tsx

```diff
@@ -225,7 +227,8 @@ const Hero = React.forwardRef<HTMLDivElement, HeroProps>(
                             template="6+6"
                             left={
                                 <Box
-                                    paddingY={noPaddingY ? 0 : 56}
+                                    paddingTop={noPaddingY ? 0 : 56}
+                                    paddingBottom={hasSlideshowBullets ? 48 : noPaddingY ? 0 : 56}
```

Comment:
In desktop the padding bottom has always 56px with bullets and without them when noPaddingY is false

Reply:
Fixed